llvm/test/CodeGen/X86/GlobalISel/x87.ll

```diff
@@ -0,0 +1,221 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple=i686-- -mattr=+x87,-sse,-sse2 -global-isel | FileCheck %s --check-prefixes=CHECK-32,GISEL_X86
+; RUN: llc < %s -mtriple=x86_64-- -mattr=+x87,-sse,-sse2 -global-isel | FileCheck %s --check-prefixes=CHECK-64,GISEL_X64
+; RUN: llc < %s -mtriple=i686-- -mattr=+x87,-sse,-sse2 | FileCheck %s --check-prefixes=CHECK-32,SDAG_X86
+; RUN: llc < %s -mtriple=x86_64-- -mattr=+x87,-sse,-sse2 | FileCheck %s --check-prefixes=CHECK-64,SDAG_X64
```

Comment:
please can you add fast-isel tests and move this file up to llvm\test\CodeGen\X86\isel-x87.ll - that's where we're putting test multi-isel test files

Reply:
I'm not sure it belongs in the top-level directory. Presumably this would be covered by an existing test there, in which case it would gain global-isel run lines. I don't think this should bother testing the fast-isel case (or the DAG case, but I could see why that one would be more useful to see whenever the output matches)